drake/systems/plants/xmlUtil.cpp

```diff
@@ -207,14 +241,14 @@ string resolveFilename(const string& filename,
   } else {
     std::string normalized_root_dir = spruce::path(root_dir).getStr();
 
-    // if root_dir is a relative path then convert it to absolute
+// if root_dir is a relative path then convert it to absolute
 #ifdef _WIN32
-    bool dirIsRelative = !(normalized_root_dir.size() >= 2
-                           && std::isalpha(normalized_root_dir[0])
-                           && normalized_root_dir[1] == ':');
+    bool dirIsRelative = !(normalized_root_dir.size() >= 2 &&
+                           std::isalpha(normalized_root_dir[0]) &&
+                           normalized_root_dir[1] == ':');
 #else
-    bool dirIsRelative = !(normalized_root_dir.size() >= 1
-                           && normalized_root_dir[0] == '/');
+    bool dirIsRelative =
+        !(normalized_root_dir.size() >= 1 && normalized_root_dir[0] == '/');
```

Comment:
@david-german-tri  and @jwnimmer-tri: I think we need to investigate the 5 whys here. Why did @patmarion's change pass CI's `clang-format` check but not my local execution of it?